doc/development.md

```diff
@@ -127,7 +127,7 @@ Note: MacOS users upgrading from an earlier release should perform a crc delete
 
 Use `npm run update-deps` or `npm-check-updates` package
 
-# Release steps
+# Release steps [OUTDATED - No more using Jenkins]
```

Comment:
Given this section is outdated, could you please completely remove this section?